pages/login.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://use.fontawesome.com/6d5bbe7895.js"></script>
+    <link rel="stylesheet" href="https://kalpaui.netlify.app/styles/main.css">
+    <link rel="stylesheet" href="../styles/main.css">
+    <link rel="stylesheet" href="../styles/auth.css">
+
+    <title>Padhaku | Login</title>
+</head>
+
+<body>
+    <header class="header--std">
+        <nav class="nav--std">
+            <ul class="header__list--align-start">
+                <!-- <li class="header__btn"><i class="fa fa-bars"></i></li> -->
+                <li class="header__title"><a href="../index.html">Padhaku</a></li>
+            </ul>
+            <div class="search-container">
+                <div class="input__container flex--inline">
```

Comment:
be consistent with your class naming structure. Either use __ or --.

Reply:
Chanchal that is called BEM. __ signify sub elements and -- signify modifier.